bangazonapi/views/product.py

```diff
@@ -248,6 +248,7 @@ def list(self, request):
         order = self.request.query_params.get('order_by', None)
         direction = self.request.query_params.get('direction', None)
         number_sold = self.request.query_params.get('number_sold', None)
+        price = self.request.query_params.get('min_price', None)
```

Comment:
`price` suggests any price. Suggest that you specify minimum somehow in the variable.

Reply:
Update the variable price to min_price